WordPress/Info.plist

```diff
@@ -447,9 +447,7 @@
 			<key>CFBundleURLName</key>
 			<string>GoogleSignIn</string>
 			<key>CFBundleURLSchemes</key>
-			<array>
-				<string>GOOGLE_LOGIN_SCHEME_ID</string>
-			</array>
```

Comment:
@jkmassel I see that Google login issue is fixed, so are these removals still expected?

Reply:
Yep, they'll land in the next beta, then I'll cut another PR to merge that fix back to `develop`!